### DISCUSSION TITLE
Website is not working

### DISCUSSION THREAD

**Discussion by marusinm** [sanitizers/octomachinery#8] [Q&A] [answered]
Is there any official documentation? It looks like your website https://octomachinery.dev is not working. Thx

**Comment by webknjaz** [comment]
Hi,

It's just a landing with a few further links now. What do you mean by saying that it does not work?

Docs are going to be here https://docs.octomachinery.dev/. I'm working on it, but no reference for now. Not yet.

But there's also some good news. You can go through a few tutorials at https://tutorial.octomachinery.dev/. They are pretty usable and should give you a basic understanding of what the UX looks like. Feel free to ask anything or file bug reports/PRs against the tutorial.

I hope this helps :)

**Comment by marusinm** [comment]
I think there is some problem with certificates as I'm stuck on `privacy error` in my browser so I can't view content of the https://octomachinery.dev and any subsite.

**Comment by webknjaz** [comment]
@marusinm oh that's weird... What browser do you use? Have you updated your OS's CA bundle recently?

**Comment by webknjaz** [comment]
This is what Google Chrome reports wrt to the cert used:
```
SSL Server Certificate
Common Name (CN)	sni.cloudflaressl.com
Organization (O)	CloudFlare, Inc.
Organizational Unit (OU)	<Not Part Of Certificate>
Common Name (CN)	CloudFlare Inc ECC CA-2
Organization (O)	CloudFlare, Inc.
Organizational Unit (OU)	<Not Part Of Certificate>
Issued On	Thursday, February 28, 2019 at 1:00:00 AM
Expires On	Friday, February 28, 2020 at 1:00:00 PM
SHA-256 Fingerprint	56 AC D7 64 1C 65 01 61 A3 72 25 5E 82 E6 08 DC
20 AA 3B 78 BE 27 20 4A 05 83 A0 32 2D EC 31 95
SHA-1 Fingerprint	9B A5 05 54 CB EE 0C 16 80 BA CC B3 A5 B0 B5 9B
CB B6 6E DA
```

It's 100% not broken and is generated/managed by CloudFlare.

Does your browser not support modern TLS? Maybe? Do you have your computer clock in sync?

**Comment by webknjaz** [comment]
@marusinm can you share what you see? Also, you can check what certificate your browser sees... It should state what it thinks is wrong.

**Comment by webknjaz** [comment, marked as answer]
Oh, and can you see errors on other `.dev` domains? Like https://docs.octomachinery.dev/ subdomain or https://github.dev/ ?
If yes, maybe your machine has `.dev` substituted for local development (which is wrong and discouraged, btw).

P.S. Everyone else I know who tried to open https://octomachinery.dev succeeded so far... So it must be something with your env. Have you tried it in other browser, machine, phone, network etc.?

**Comment by webknjaz** [comment]
I'm going to come to [Brněnské Pyvo](https://pyvo.cz/en/brno-pyvo/2019-03/) today. If you're around, find me there and I'll help you figure this out :)

**Comment by marusinm** [comment]
> maybe your machine has .dev substituted for local development 

You were absolutely right, that was beginner's mistake on my side. Thx, solved.
